### PR TITLE
Create update.json

### PR DESCRIPTION
**Pull request by tom2strobl**

### CODE REVIEW

**Comment by pnommensen** [comment]
@tom2strobl: The non-minified file doesn't seem necessary for production use. If this is the case, please update the update.json accordingly. Thx.

**Comment by tom2strobl** [comment]
Updated both files to be in the same PR and excluded the non-minified version as you're right, nobody will really need it.

**Comment by tom2strobl** [comment]
PR #4613 is probably now to be closed, right?

File: files/audiofx/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Thomas Strobl"
+github = "https://github.com/madebywild/audioFX"
+homepage = "https://github.com/madebywild/audioFX"
+description = "High-Level Audio Effects using the Web Audio API in JavaScript, ~1-2 KB gzipped."
+mainfile = "dist/AudioFX.min.js"
```
Comment (jimaek) [inline files/audiofx/info.ini]: Just `AudioFX.min.js` is the correct

**Comment by jimaek** [comment]
Once you fix the problems please either squash your commits or open a new PR. The maximum amount of commits per PR is 2

**Comment by tom2strobl** [comment]
Closed and opened #4628

**Pull request closed without merging**